solutions/cf-888G.mdx

```diff
@@ -0,0 +1,162 @@
+---
+id: cf-888G
+source: CF
+title: Xor-MST
+author: Dustin Miao
+---
+
+## Solution
+
+The [Official Editorial](https://codeforces.com/blog/entry/55701) uses a solution with Boruvka's Algorithm. There is a simpler solution using divide and conquer.
+
+Firstly, note that because there is essentially an edge between any pair of nodes, the value of the nodes themselves can be considered unordered. Thus, the answer will not change if we sort the elements. 
```

Comment:
```suggestion
Firstly, note that because there is essentially an edge between any pair of nodes, the values of the nodes themselves can be considered unordered. Thus, the answer will not change if we sort the elements. 
```